sparkles/tests/test_checks.py

```diff
@@ -80,6 +80,66 @@ def test_n_guide_check_not_enough_stars():
     ]
 
 
+def test_or_too_cold():
```

Comment:
DRY.

Rewrite this using `pytest.mark.parametrize("case", cases)` where `cases` is a list of dict with the relevant inputs and expected `messages`. Get used to using `parametrize` for a number of test cases that are variations on a theme.

Reply:
Well, I suppose in this new era DRY is "encourage copilot not to repeat itself when writing tests".

Reply:
And that can be done by training it with good examples in your own code. :smile: